README.md

```diff
@@ -2,5 +2,38 @@
 
 ## General
 
-ClickHouse connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and
-a destination ClickHouse connector.
\ No newline at end of file
+ClickHouse connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides a destination
+ClickHouse connector.
+
+## Prerequisites
+
+- [Go](https://go.dev/) 1.18
+- [Golang SQL database client for ClickHouse](https://github.com/ClickHouse/clickhouse-go) 2.3.0
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.49.0
+- (optional) [mock](https://github.com/golang/mock) 1.6.0
+
+## How to build it
+
+Run `make build`.
+
+## Testing
+
+Run `make test` to run all unit and integration tests. To pass the integration test, set the ClickHouse database URL to
+the environment variables as an `CLICKHOUSE_URL`.
+
+## Destination
```

Comment:
It would be good to have some notes about how we use record keys.

Reply:
Also, it would be good to have some general notes about how we handle errors (do we fail fast, do we retry and if so, in which way etc.)

Reply:
I've added an additional tab with `Key Handling`.

And about how we handle errors. We simply return the errors that occurred. Without retries. That's why I'm not sure this information is worth mentioning.

Reply:
> And about how we handle errors. We simply return the errors that occurred. Without retries. That's why I'm not sure this information is worth mentioning.

It would be good to check if there are any errors which can be retried or which we can fix ourselves.

Reply:
Info from @voscob during a call: The retries are done in the driver, so no need to do those in the connector.